polling_stations/apps/data_collection/management/commands/import_south_bucks.py

```diff
@@ -0,0 +1,49 @@
+from data_collection.management.commands import BaseXpressDemocracyClubCsvImporter
+
+
+class Command(BaseXpressDemocracyClubCsvImporter):
+    council_id = "E07000006"
+    addresses_name = (
+        "parl.2019-12-12/Version 1/Democracy_Club__12December2019 Beaconsfield.tsv"
+    )
+    stations_name = (
+        "parl.2019-12-12/Version 1/Democracy_Club__12December2019 Beaconsfield.tsv"
+    )
+    elections = ["parl.2019-12-12"]
+    csv_delimiter = "\t"
+    # allow_station_point_from_postcode = False
+
+    def station_record_to_dict(self, record):
+        return super().station_record_to_dict(record)
+
+    def address_record_to_dict(self, record):
+        if (
+            record.addressline1 == "Pilgrim House"
+            and record.addressline2 == "12H Packhorse Road"
+        ):
+            record = record._replace(property_urn="", addressline6="SL9 7QE")
+
+        if record.addressline6 == "SL0 0AF":
+            return None  # looks odd
+
+        rec = super().address_record_to_dict(record)
+        uprn = record.property_urn.strip().lstrip("0")
+
+        if uprn in [
+            "100081076380",  # SL23HW -> SL23HJ : Brook House, Templewood Lane, Stoke Poges, Slough
+            "10003236726",  # UB94DQ -> UB94DG : Flat 4 The Old Mill, Oxford Road, Denham, Uxbridge
+            "10003236727",  # UB94DQ -> UB94DG : Flat 6 The Old Mill, Oxford Road, Denham, Uxbridge
+            "10003236729",  # UB94DQ -> UB94DG : Flat 8 The Old Mill, Oxford Road, Denham, Uxbridge
```

Comment:
You sure about these 3? Have a look at them on `dashboard/postcode/UB94DG/` and `dashboard/postcode/UB94DQ/`

Reply:
Not now you mention it ;-). Now noped.